java/ql/lib/ext/org.springframework.jdbc.core.namedparam.model.yml

```diff
@@ -0,0 +1,15 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["org.springframework.jdbc.core.namedparam", "NamedParameterJdbcOperations", True, "batchUpdate", "", "", "Argument[0]", "sql", "manual"]
+      - ["org.springframework.jdbc.core.namedparam", "NamedParameterJdbcOperations", True, "batchUpdate", "(String[])", "", "Argument[0]", "sql", "manual"]
```

Comment:
I started by copy-pasting the [`JdbcTemplate` models](https://github.com/github/codeql/blob/1b06bf132c7962ace57277a99a3ca5d9e3adaecd/java/ql/lib/ext/org.springframework.jdbc.core.model.yml#L7) because of their similarity, and I forgot to remove that one. Thanks for catching it! 🙇

Reply:
```suggestion
```